concord/win_docker_ts_01.ps1

```diff
@@ -0,0 +1,3 @@
+Write-Host "Running Admin elevated PS script [$args\win_docker_ts_02.ps1]"
```

Comment:
We would probably need to add this to k8s-cluster script in builder projects. As this needs to by synced each time we would like to submit some process to concord.